src/solid-shapes/include/iDynTree/InertialParametersSolidShapesHelpers.h

```diff
@@ -32,8 +33,26 @@ namespace iDynTree
 bool estimateInertialParametersFromLinkBoundingBoxesAndTotalMass(const double totalMass,
                                                                  iDynTree::Model& model,
                                                                  VectorDynSize& estimatedInertialParams);
+/**
+ * @brief Compute bounding box from a solid shape object
+ *
+ * @param[in] geom The link collision as a iDynTree solid shape object
+ * @param[out] box The bounding box for the solid shape object
+ * @return true if all went well, false otherwise .
+ *
+ * @note If the shape is an ExternalMesh type, this function requires IDYNTREE_USES_ASSIMP to be true, otherwise it always returns false.
+ * @note If the shape is an ExternalMesh type, an Axis-Aligned Bounding Box (AABB) is computed from the mesh vertices. 
+ *       This means that the returned bounding box is neither unique nor a minimum volume bounding box for the given mesh.
+ */
+bool computeBoundingBoxFromShape(const SolidShape* geom, Box& box);
```

Comment:
Sorry, I meant to use a reference:
```suggestion
bool computeBoundingBoxFromShape(const SolidShape& geom, Box& box);
```

Reply:
I had misread the symbol. I am fixing it.

Reply:
Fixed in https://github.com/robotology/idyntree/pull/801/commits/299c67d8d7ac0d6b11148ea6d6d35b2c959e8711